charts/hermes/templates/secret.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "hermes.fullname" . }}-env
+  labels:
+    {{- include "hermes.labels" . | nindent 4 }}
+data:
+  s3-access-key-id: {{ required "A valid S3 access key id is required" .Values.secrets.s3AccessKeyId | b64enc }}
+  s3-access-secret-key: {{ required "A valid S3 access secret key is required" .Values.secrets.s3AccessSecretKey | b64enc }}
```

```suggestion
data:
{{- if and (eq .Values.config.stream.type "s3") .Values.config.stream.s3.enabled }}
  s3-access-key-id: {{ required "A valid S3 access key id is required" .Values.secrets.s3AccessKeyId | b64enc }}
  s3-access-secret-key: {{ required "A valid S3 access secret key is required" .Values.secrets.s3AccessSecretKey | b64enc }}
{{-end }}
```

Maybe something like this?

---

Ahhh yes. Good spot, tyty. Updated [5da2139](https://github.com/ethpandaops/ethereum-helm-charts/pull/356/commits/5da2139a2e163f1f091282774752072581323a54) @samcm